JetStreamDriver.js

```diff
@@ -2313,6 +2325,8 @@ for (const benchmark of BENCHMARKS) {
         throw new Error(`Duplicate benchmark with name "${name}}"`);
     else
         benchmarksByName.set(name, benchmark);
+    
+    benchmark.tags.add("all");
 
     for (const tag of benchmark.tags) {
```

Comment:
You probably want to `tag = tag.toLowerCase()` this before checking/inserting it into the map.

Reply:
The conversion is happening in the Benchmark constructor already.
I plan to add some basic unittests for this in a separate PR so we don't accidentally mess this up.